source/dpp/translation/translation.d

```diff
@@ -257,7 +257,8 @@ bool untranslatable(in string line) @safe pure {
         || line.canFind("variant!")
         || line.canFind("value _ ")
         || line.canFind("enable_if_c")
-        || line.canFind(`}))`)  // ???
+        // causing trouble with gcc __extension__ (#168)
+        // || line.canFind(`}))`)  // ??? 
```

Comment:
If this can be deleted without breaking anything, please do that instead of commenting it out.

Reply:
Updated ldc2 to 1.15... Retesting.

Reply:
All tests passed :-)